docs/deploying.md

```diff
@@ -22,17 +22,23 @@ However, the kit won't show your prototype online without setting a username and
 
 ### 3) Set a username and password
 
-You can do this in the Heroku admin console (under Settings ▶ Config Variables) or by running the following commands:
+You can do this in the on the Heroku website (under Settings ▶ Config Variables) or by running the following commands:
 
 ```
 heroku config:set USERNAME=username_here
 heroku config:set PASSWORD=password_here
 ```
-
 If you don't want to have password protection on your prototype, you can set the `USE_AUTH` config var:
+
 ```
 heroku config:set USE_AUTH=false
 ```
+If you have more than one remote you'll need to add a flag `-r 'remote_name'` to specifiy which remote to set.
+
+```
+heroku config:set PASSWORD=password_here -r dev
```

Comment:
maybe we should use the same style as the other examples:

heroku config:set PASSWORD=password_here -r branchname_here